pages/📊 Stable curve.py

```diff
@@ -59,118 +53,115 @@
+      df["New curve"] = df[x_data_name].apply(lambda x: new_stable_swape.calculate_y(type_token_sell, y_data["name"], x))
+
+      df["Current spot price"] = df[x_data_name].apply(lambda x: current_stable_swape.calculate_spot_price(type_token_sell, y_data["name"], x))
+      df["New spot price"] = df[x_data_name].apply(lambda x: new_stable_swape.calculate_spot_price(type_token_sell, y_data["name"], x))
+
+
+      fig = go.Figure(go.Scatter(
+          name = "Default Amp Factor",
+          x = df[x_data_name],
+          y = df["Current curve"],
+          hovertemplate =
+          '<b>Price: %{text}</b>',
+          text = df["Current spot price"].map('{:.5f}'.format)))
+
+      fig.add_trace(go.Scatter(
+          name = "New Amp Factor",
+          x = df[x_data_name],
+          y = df["New curve"],
+          hovertemplate = '<b>Price: %{text}</b>',
+          text = df["New spot price"].map('{:.5f}'.format)))
+
+      fig.update_layout(hovermode='x unified')
+
+      type_token_buy = y_data["name"]
+      new_transaction = new_stable_swape.calculate_trade(type_token_sell, y_data["name"], amount_token_sell)
+      current_transaction = current_stable_swape.calculate_trade(type_token_sell, y_data["name"], amount_token_sell)
+      new_amount_token_buy = new_transaction['amount_token_buy']
+
+      current_amp_dict = {'amp':base_amp, 'type_token_buy': y_data["name"], 'amount_token_buy':current_transaction['amount_token_buy'], 'price':current_transaction['price'], 'default_price':current_transaction['price'] }
+      new_amp_dict = {'amp':new_amp, 'type_token_buy': y_data["name"], 'amount_token_buy':new_transaction['amount_token_buy'], 'price':new_transaction['price'], 'default_price':current_transaction['price'] }
+
+      html_components.amp_price_conteiner(current_amp_dict, new_amp_dict, type_token_sell)
+
+      #todo find a better way to to this hover
+      hover = "<br>%{text}<br>"
+      hover += "{}: ".format(st.session_state["x_data"]["name"])
+      hover += "<b>%{x}</b><br>"
+      hover += "{}: ".format(y_data["name"])
+      hover += "<b>%{y}</b><br>"
+
+      st.write('You paid',new_transaction['price'], y_data["name"], 'for 1',  st.session_state["x_data"]["name"])
+      fig.add_scatter(mode="markers",x=current_transaction['transaction_sell'],y=current_transaction['transaction_buy'], text=current_transaction['label'],name="", hovertemplate=hover,         
         marker=dict(
             color='#2533F8',
             size=7,
         ), showlegend=False)
-      pool_fig.add_scatter(mode="markers",x=new_transaction['transaction_buy'],y=new_transaction['transaction_sell'], text=new_transaction['label'],name="", hovertemplate='%{x} ; %{y}',         
+      fig.add_scatter(mode="markers",x=new_transaction['transaction_sell'],y=new_transaction['transaction_buy'], text=new_transaction['label'],name="",hovertemplate=hover,         
```

Comment:
x and y on hover are not needed anymore

Reply:
fix on [commit](https://github.com/luizakp/balancer-stable-streamlit/pull/17/commits/05eb4efed4064773a1d9147d0e0fa4b267c2857c)